openapi/components/schemas/Customer.yaml

```
@@ -100,8 +100,8 @@ properties:
     default: false
   hasFulfilledKyc:
     description: >-
-      Whether the customer has fulfilled a KYC request by providing valid
-      identity and address documents.
+      True when the customer has accepted and reviewed identity and address documents
+      or an accepted credit file document.
```

nitpick: I think it's better to start descriptions with an active verb

```suggestion
      Specifies whether the customer has accepted and reviewed identity and address documents,
      or an accepted credit file document.
```